### PR TITLE
Fix footer

### PR DESCRIPTION
**Pull request by aman2712**

### CODE REVIEW

**Comment by ShantanuLamba** [comment]
Our theme is black not white ig and the footer should drag down (maybe add some margin)

**Comment by ShantanuLamba** [comment]
Everything is fine, I reviewed

**Comment by aman2712** [comment]
> Our theme is black not white ig and the footer should drag down on the create challenges page(maybe add some margin)

It will automatically come down when you add a section on top of it

File: frontend/src/components/Footer.vue [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
-        <b-col>
+        </div>
+        <div class="my-2 my-md-0 mb-4">
+          <h2>Contact Us</h2>
+          <div class="d-flex align-items-center">
+            <b-icon-envelope />&nbsp;
+            <b-link class="d-block text-light">prograchallenges@gmail.com</b-link>
+          </div>
+          <div class="d-flex align-items-center">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-telegram" viewBox="0 0 16 16">
```
Comment (Wittmaxi) [inline frontend/src/components/Footer.vue:21]: Please import the SVGs as a file - we might want to reuse the icons + it makes the HTML-code easier to read

**Review by Wittmaxi — CHANGES_REQUESTED** [review]
Pretty good, please export the SVGs and merge!

**Pull request merged**